gbdk-lib/examples/cross-platform/platformer_template/Makefile

```diff
@@ -0,0 +1,129 @@
+#
+# A Makefile that compiles all .c and .s files in "src" and "res" 
+# subdirectories and places the output in a "obj" subdirectory
+#
+
+# If you move this project you can change the directory 
+# to match your GBDK root directory (ex: GBDK_HOME = "C:/GBDK/"
+ifndef GBDK_HOME
+	GBDK_HOME = ../../../
+endif
+	
+
+PNG2ASSET = $(GBDK_HOME)bin/png2asset 
+LCC = $(GBDK_HOME)bin/lcc 
+
+# Set platforms to build here, spaced separated. (These are in the separate Makefile.targets)
+# They can also be built/cleaned individually: "make gg" and "make gg-clean"
+# Possible are: gb gbc pocket megaduck sms gg
+TARGETS=gb pocket megaduck sms gg nes
+
+# You can set flags for LCC here
+# For example, you can uncomment the line below to turn on debug output
+# LCCFLAGS = -debug
+
+# You can set the name of the .gb ROM file here
+PROJECTNAME    = platformer_template
+
+SRCDIR      = src
+DISTDIR      = dist
+OBJDIR      = obj/$(EXT)
+RESDIR      = res
+GENDIR      = gen/$(EXT)/src
+BINDIR      = build/$(EXT)
+MKDIRS      = $(GENDIR) $(OBJDIR) $(BINDIR) # See bottom of Makefile for directory auto-creation
+BINS	    = $(OBJDIR)/$(PROJECTNAME).$(EXT)
+CSOURCES    = $(foreach dir,$(SRCDIR),$(notdir $(wildcard $(dir)/*.c))) 
+GENSOURCES    = $(foreach dir,$(GENDIR), $(wildcard $(dir)/*.c))
+ASMSOURCES  = $(foreach dir,$(SRCDIR),$(notdir $(wildcard $(dir)/*.s)))
+OBJS       = $(CSOURCES:%.c=$(OBJDIR)/%.o) $(ASMSOURCES:%.s=$(OBJDIR)/%.o)
+
+LCCFLAGS += -Wl-j -Wm-yoA -Wm-ya4 -Wb-ext=.rel -Wb- $(LCCFLAGS_$(EXT)) # This adds the current platform specific LCC Flags
```

Comment:
There is a stray `-Wb- ` at the end causing bankpack to issue a warning due to the detached ` - `. Probably just remove it.

"../../../"bin/bankpack -plat=nes -mapper=30 -ext=.rel - -yt0x1B -lkout=/tmp/lcc2393229.lk obj/nes/camera.o obj/nes/common.o obj/nes/level.o obj/nes/main.o obj/nes/player.o /tmp/lcc2393220.o /tmp/lcc2393221.o /tmp/lcc2393222.o /tmp/lcc2393223.o /tmp/lcc2393224.o /tmp/lcc2393225.o /tmp/lcc2393226.o /tmp/lcc2393227.o /tmp/lcc2393228.o
BankPack: Warning: Ignoring unknown option -